pkg/bufferdecoder/protocol_test.go

```diff
@@ -46,5 +46,5 @@ func TestBpfObjectMetaSize(t *testing.T) {
 func TestMprotectWriteMetaSize(t *testing.T) {
 	var v MprotectWriteMeta
 	size := int(unsafe.Sizeof(v))
-	assert.Equal(t, size, int(v.GetSizeBytes()))
+	assert.Equal(t, size-4, int(v.GetSizeBytes()))
```

Comment:
can you please explain this change?

Reply:
There is alignment so we fix that, you can see above on other examples that are not 8byte aligned.